test/test_gemqrt.cc

```diff
@@ -0,0 +1,189 @@
+{
+    return LAPACKE_sgemqrt(
+        LAPACK_COL_MAJOR, side, trans, m, n, k, nb,
+        V, ldv,
+        T, ldt,
+        C, ldc );
+}
+
+inline lapack_int LAPACKE_gemqrt(
+    char side, char trans, lapack_int m, lapack_int n, lapack_int k, lapack_int nb,
+    double* V, lapack_int ldv,
+    double* T, lapack_int ldt,
+    double* C, lapack_int ldc )
+{
+    return LAPACKE_dgemqrt(
+        LAPACK_COL_MAJOR, side, trans, m, n, k, nb,
+        V, ldv,
+        T, ldt,
+        C, ldc );
+}
+
+inline lapack_int LAPACKE_gemqrt(
+    char side, char trans, lapack_int m, lapack_int n, lapack_int k, lapack_int nb,
+    std::complex<float>* V, lapack_int ldv,
+    std::complex<float>* T, lapack_int ldt,
+    std::complex<float>* C, lapack_int ldc )
+{
+    return LAPACKE_cgemqrt(
+        LAPACK_COL_MAJOR, side, trans, m, n, k, nb,
+        (lapack_complex_float*) V, ldv,
+        (lapack_complex_float*) T, ldt,
+        (lapack_complex_float*) C, ldc );
+}
+
+inline lapack_int LAPACKE_gemqrt(
+    char side, char trans, lapack_int m, lapack_int n, lapack_int k, lapack_int nb,
+    std::complex<double>* V, lapack_int ldv,
+    std::complex<double>* T, lapack_int ldt,
+    std::complex<double>* C, lapack_int ldc )
+{
+    return LAPACKE_zgemqrt(
+        LAPACK_COL_MAJOR, side, trans, m, n, k, nb,
+        (lapack_complex_double*) V, ldv,
+        (lapack_complex_double*) T, ldt,
+        (lapack_complex_double*) C, ldc );
+}
+
+//------------------------------------------------------------------------------
+template< typename scalar_t >
+void test_gemqrt_work( Params& params, bool run )
+{
+    using real_t = blas::real_type< scalar_t >;
+    typedef long long lld;
+
+    // get & mark input values
+    lapack::Side side = params.side();
+    lapack::Op trans = params.trans();
+    int64_t m = params.dim.m();
+    int64_t n = params.dim.n();
+    int64_t k = params.dim.k();
+    int64_t nb = params.nb();
+    int64_t align = params.align();
+
+    // mark non-standard output values
+    params.ref_time();
+    params.ref_gflops();
+    params.gflops();
+
+    if (! run)
+        return;
+
+    //---------- setup
+    int64_t ldv;
+    if (side == lapack::Side::Right) {
+        ldv = roundup( blas::max( 1, m ), align );
+    } else {
+        ldv = roundup( blas::max( 1, n ), align );
+    }
+    int64_t ldt = roundup( nb, align );
+    int64_t ldc = roundup( blas::max( 1, m ), align );
+    size_t size_V = (size_t) ldv * k;
+    size_t size_T = (size_t) ldt * k;
+    size_t size_C = (size_t) ldc * n;
+
+    std::vector< scalar_t > V( size_V );
+    std::vector< scalar_t > T( size_T );
+    std::vector< scalar_t > C_tst( size_C );
+    std::vector< scalar_t > C_ref( size_C );
+
+    int64_t idist = 1;
+    int64_t iseed[4] = { 0, 1, 2, 3 };
+    lapack::larnv( idist, iseed, V.size(), &V[0] );
+    lapack::larnv( idist, iseed, T.size(), &T[0] );
+    lapack::larnv( idist, iseed, C_tst.size(), &C_tst[0] );
+    C_ref = C_tst;
+
```

Comment:
You will need to add a call here to `lapack::geqrt` before calling `gemqrt` in order to setup the matrices, before running the test.

Reply:
Got it, fixed. 
I still have the actual test call commented out; it is likely failing because i am setting up the line in run_test.py wrong.

Reply:
Fixed.